azkaban-common/src/main/java/azkaban/scheduler/Schedule.java

```diff
@@ -96,6 +101,31 @@ public Schedule(int scheduleId, int projectId, String projectName,
     this.submitTime = submitTime;
     this.executionOptions = executionOptions;
     this.slaOptions = slaOptions;
+    this.cronExpression = cronExpression;
+  }
+
+  /**
+   * Constructor for Cron Scheduler
```

Comment:
Is the intention to have one constructor for the old period based schedule and a new one for the Cron based schedule?

Is it possible to call the other constructor internally which has all the options?